common/static/common/js/discussion/views/discussion_thread_list_view.js

```diff
@@ -335,7 +336,14 @@
             DiscussionThreadListView.prototype.renderThread = function(thread) {
                 var threadCommentCount = thread.get('comments_count'),
                     threadUnreadCommentCount = thread.get('unread_comments_count'),
-                    neverRead = !thread.get('read') && threadUnreadCommentCount === threadCommentCount,
+                    // @TODO: On the profile page, thread read state for the viewing user is not accessible via the API.
```

Comment:
I recommend moving this comment to the profile page itself. I don't think it will be necessary if the variable is renamed to be more functional and not page specific.